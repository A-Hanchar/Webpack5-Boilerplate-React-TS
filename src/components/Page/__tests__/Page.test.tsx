import { render } from '@testing-library/react'
import React from 'react'

import { Page } from '../'

describe('<Page />', () => {
  it('SHOULD match snapshot', () => {
    const { asFragment } = render(<Page />)

    expect(asFragment()).toMatchSnapshot()
  })
})
