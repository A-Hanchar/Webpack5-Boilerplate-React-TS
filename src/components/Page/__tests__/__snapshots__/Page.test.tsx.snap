// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> SHOULD match snapshot 1`] = `
<DocumentFragment>
  <div
    class="root"
  />
</DocumentFragment>
`;
